src/storage/test/StorageIndexWriteBenchmark.cpp

```diff
@@ -161,34 +261,37 @@ void insertVertices(bool withoutIndex) {
         }
```

Comment:
line 142 ~ line 161
This part of the code can be proposed separately
Avoid code duplication and Can be reused